languages/README.md

```diff
@@ -0,0 +1,134 @@
+# Instructions for CloudEvents Specification in Multi-languages
+
+<!-- no verify-specs -->
+
+## Abstract
+
+This page contains information about how to translate a document from scratch, 
+definition of roles and responsibilities as well as guidelines that translators 
+and reviewers should follow.
+
+## Status of this document
+
+This is a working draft. Changes could be made anytime.
+
+## Table of Contents
+
+- [Roles](#roles)
+- [How to translate](#how-to-translate)
+- [How to add a new language](#how-to-add-a-new-language)
+- [How to publish translation tasks](#how-to-publish-translation-tasks)  
+- [Translation style guide](#translation-style-guide)
+
+## Roles
+
+There are two categories of roles:
+
+1. **Translator.** This is anyone who participates in the translation of any 
+   CloudEvents documents. One becomes a formal translator once a Pull Request (PR) 
+   of their translation is merged.
+
+2. **Reviewer.** A reviewer is responsible for checking new translations 
+   and answering related issues.
+   Typically, reviewers of each language are members of a Github team which is named 
+   after the language code(i.e., zh-CN reviewers).
+
+### Translators
+
+A translator can start translating by following [the translation instructions](#how-to-translate).
+Translators should follow the [translation style guide](#translation-style-guide).
+
+### Reviewers
+
+Anyone, who builds [a new language structure](#how-to-add-a-new-language), will be the first reviewer
+of that language.
+
+Reviewers should track translation state of each document by editing 
+`languages/{language-code}/translations.md` file.
```

Comment:
Is it going to be a pain to have to submit a PR to edit this doc? I'm wondering if we should use a google doc and then we can easily add people to give them edit access.

Reply:
actually, I guess you're assuming that they'll edit this doc in the same PR as the newly edited files, right? If so, ignore my comment :-)